docs/DeveloperGuide.adoc

```diff
@@ -242,9 +265,56 @@ _{Explain here how the data encryption feature will be implemented}_
 
 // end::dataencryption[]
 
+// tag::sorttransactions[]
+=== Sort Transactions Feature: `sort`
+
+This feature allows the user to _sort_ their bank account by the _amount_ or _date_ of their transactions by using the command `sort amount` or `sort date` respectively.
+
+==== Current Implementation
+
+The `sort` command is facilitated by the _Logic_ and _Model_ components of the application, _PalPay_.
+
+The following sequence diagram shows how the sorting of transactions work when the user enters `sort amount`.
+
+image::SortSequenceDiagram.png[]
+Figure [Fill in Later].
+Sequence Diagram for `sort amount`
+
+-To Be Continued-
```

Comment:
![image](https://user-images.githubusercontent.com/43642522/67361596-be1fc580-f59b-11e9-8fd3-12cc0f01e81b.png)

I think the above implementation sort of breaks Separation of Concerns Principle, as the `BankAccount` is now required to do the work of creating and returning a sorted transaction history. Maybe a better implementation is to return the normal transaction history, and sort it inside the `SortCommand`?